modules/web-console-overview.adoc

```diff
@@ -20,3 +17,5 @@ INFO Login to the console with user: kubeadmin, password: <provided>
 ----
 
 Use those details to log in and access the web console.
+
+For existing clusters that you did not install, you can use `oc whoami --show-console` to locate the login credentials.
```

Comment:
This doesn't actually locate the credentials. It only prints the URL of the console.